### PR TITLE
Add RSS feed

### PR DESCRIPTION
**Pull request by delta1**
Add RSS feed as per issue #1999

### CODE REVIEW

**Comment by delta1** [comment]
Thanks for the feedback @lex111 - I did accept the changes however I have not tested them locally

**Comment by delta1** [comment]
Tested locally and working perfectly 👍

**Comment by burden** [comment]
This definitely looks like an rss/atom feed. Has any one had a chance to checkout the [jekyll-feed plugin](https://github.com/jekyll/jekyll-feed)?

**Comment by doomspork** [comment]
@burden as the Jekyll master I'll defer to you on this PR and the plugins. Happy to weigh in though if need be 😁

File: feed.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    xmlns:atom="http://www.w3.org/2005/Atom">
+    <channel>
+        <title>{{ project.title }}</title>
+        <description>{{ project.description }}</description>
+        <link>{{ site.url }}</link>
+        <atom:link href="{{ site.url }}/feed.xml" rel="self" type="application/rss+xml" />
+    {% for post in site.posts %}
+        <item>
+            <title>{{ post.title | xml_escape }}</title>
+            <description>{{ post.content | strip_html | truncatewords:75 }}</description>
```
[the remark this answers is not in the document]
Comment (delta1) [reply feed.xml]: Thanks @burden - done

**Pull request merged**